str/trtools/merge_str_prep.py

```diff
@@ -4,7 +4,7 @@
 This script prepares GangSTR/EH VCF files for input into mergeSTR.
 Required input: --caller, --input-dir, and external sample IDs
 For example:
-analysis-runner --access-level test --dataset tob-wgs --description 'tester' --output-dir 'str/5M_run_combined_vcfs/merge_str_prep' merge_str_prep.py --caller=eh --input-dir=gs://cpg-tob-wgs-test/str/5M_run_combined_vcfs CPGtestersorted2
+analysis-runner --access-level test --dataset tob-wgs --description 'tester' --output-dir 'str/5M_run_combined_vcfs/merge_str_prep/v2' merge_str_prep.py --caller=eh --input-dir=gs://cpg-tob-wgs-test/hoptan-str --sharded sharded_tester
```

Comment:
Could make this more readable:

```suggestion
analysis-runner \
    --access-level test \
    --dataset tob-wgs \
    --description 'tester' \
    --output-dir 'str/5M_run_combined_vcfs/merge_str_prep/v2' \
    merge_str_prep.py \
        --caller=eh \
        --input-dir=gs://cpg-tob-wgs-test/hoptan-str \
        --sharded sharded_tester
```